File: Python/lab10/Administrator.py
from Worker import Worker
from WorkerDecorator import WorkerDecorator

class Administrator(WorkerDecorator):
    def __init__(self, worker: Worker):
        super().__init__(worker)

    def makeJob(self) -> str:
        return " Take interviews and keep track of everything"

    def doWork(self) -> str:
        return super().doWork() + self.makeJob()

File: Java/lab10/src/main/java/org/example/Administrator.java
package org.example;

public class Administrator extends WorkerDecorator {
    public Administrator(Worker worker) {
        super(worker);
    }
    public String makeJob(){
        return " Take interviews and keep track of everything";
    }
    @Override
    public String doWork() {
        return super.doWork()+makeJob();
    }
}
